data/json/items/migration.json

```diff
@@ -204,6 +204,26 @@
     "replace": "reloaded_shot_slug"
   },
   {
+      "id": "fish_bait_meat",
```

Comment:
I think you pay way too much attention to that but wev, I can update this easily.